src/main/java/org/assertj/core/error/GroupTypeDescription.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2020 the original author or authors.
+ */
+package org.assertj.core.error;
+
+import java.util.Map;
+
+/**
+ * Generates a description for the type of a group of elements. The description is used in the error message.
+ *
+ */
+public class GroupTypeDescription {
```

Comment:
add a `GroupTypeDescription_getGroupTypeDescription_Test` test for this class to check `getGroupTypeDescription` with a parameterized test (see https://www.baeldung.com/parameterized-tests-junit-5#6-method)